lntest/itest/lnd_test.go

```diff
@@ -2014,8 +2024,14 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob didn't report channel: %v", err)
 	}
 
-	// Create Carol and a new channel Bob->Carol.
-	carol, err := net.NewNode("Carol", nil)
+	// Create Carol with options to rate limit channel updates up to 1 per
+	// day, and crate a new channel Bob->Carol.
+	carol, err := net.NewNode(
+		"Carol", []string{
+			"--gossip.max-channel-update-burst=1",
```

Comment:
We can improve the test further by using a slightly higher burst value (say 2) and asserting that the burst allotment is indeed observed (so a peer can send 2 over, but then their next update isn't propagated).